pkg/pod/pod.go

```diff
@@ -705,6 +705,25 @@ func (builder *Builder) WithSecondaryNetwork(network []*multus.NetworkSelectionE
 	return builder
 }
 
+// WithHostNetwork applies HostNetwork to pod's definition.
+func (builder *Builder) WithHostNetwork() *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Defining pod's HostNetwork")
```

Comment:
Please change logs to 
glog.V(100).Infof("Applying HostNetwork flag to pod's %s configuration", builder.Definition.Name)